scanner/src/funTest/kotlin/scanners/ScanPathTest.kt

```diff
@@ -24,8 +24,7 @@ import com.here.ort.utils.safeDeleteRecursively
 import com.here.ort.utils.test.ExpensiveTag
```

Comment:
-use plural form for function: "make some previously protected properties / function public,"
-is 'load' intended? : "with unknown download load time"
-consider removing '.' and / or ',' from the end of the listing items.

Reply:
Actually, it should have been singular for "properties", fixed.

Reply:
Done.